pkgs/by-name/dc/dc3dd/package.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchzip
+, perl538Packages
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dc3dd";
+  version = "7.3.1";
+
+  src = fetchzip {
+      url = "http://downloads.sf.net/sourceforge/dc3dd/dc3dd-${finalAttrs.version}.zip";
+      hash = "sha256-SYDoqGlsROHX1a0jJX11F+yp6CeFK+tZbYOOnScC6Ig=";
+  };
+
+  postPatch = ''
+    chmod +x configure
+  '';
+
+  buildInputs = [ perl538Packages.LocaleGettext ];
+
+  buildPhase = ''
+    runHook preBuild
+    if stdenv.hostPlatform.isi686 && stdenv.hostPlatform.isx86_64
+    then
+      ./configure --prefix=$out
+    else
+      ./configure --build=arm --prefix=$out
+    fi
+    make
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    make PREFIX=$out install
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A patched version of dd that includes a number of features useful for computer forensics";
+    mainProgram = "dc3dd";
+    homepage = "http://sourceforge.net/projects/dc3dd";
+    maintainers = with maintainers; [ d3vil0p3r ];
+    platforms = with platforms; linux;
+    license = licenses.gpl2;
```

According to the [file header](https://sourceforge.net/p/dc3dd/code/HEAD/tree/src/dc3dd.c#l6) is the license `GPL3+`.

```suggestion
    license = licenses.gpl3Plus;
```